qutip/core/cy/qobjevo.pyx

```diff
@@ -1160,6 +1174,9 @@ cdef class _Expect:
             f"does not match the state ({state.shape})."
         )
 
+    def __repr__(self):
+        return "Expect"
```

Comment:
```suggestion
        return "Expect"
```
These `__repr__`s make some sense when included in the `__repr__` of `QobjEvo` but they are very strange on their own. If we're making these names visible as we seem to be doing, we either make them clearer and document what they are.

Perhaps we can rename the classes to `_ExpectFeedback`, `_QobjFeedback` and `_DataFeedback` and then change the reprs to `ExpectFeedbak`, `QobjFeedback` and `DataFeedback`?

I'm not completely happy with my suggested names, so better suggestions welcome.